db/repos/accounts.js

```diff
@@ -42,17 +42,15 @@ const normalFields = [
 	{ name: 'rewards', cast: 'bigint', def: '0', skip: ifNotExists },
 	{ name: 'vote', cast: 'bigint', def: '0', skip: ifNotExists },
 	{
-		name: 'producedblocks',
```

Comment:
consistency - all other are 1-lines

Reply:
Afaik, this is consistent formatting behavior according to prettier, therefore we should not worry about it.